Cargo.toml

```diff
@@ -12,4 +12,6 @@ members = [
     "crates/ethernet-interface-types",
     "crates/eth-driver",
     "crates/eth-client",
+    "crates/timer",
+    "crates/timer-interface-types"
```

Comment:
Is it better to make this as a sub-crate of `timer` like `pl011-driver` does (I.e., place it in `crates/timer/interface-types/`)? I could go either way, but we should probably pick a convention.

Reply:
I moved all interfaces to `interfaces` folder